packages/server/billing/helpers/processInvoiceItemHook.ts

```diff
@@ -103,17 +103,23 @@ const processInvoiceItemHook = async (stripeSubscriptionId: string) => {
     .run()
 
   if (isFlushed) {
-    const orgUserCount = await r
+    const orgUsers = await r
       .table('OrganizationUser')
       .getAll(orgId, {index: 'orgId'})
       .filter({
         inactive: false,
         removedAt: null
       })
-      .count()
       .run()
+    const orgUserCount = orgUsers.length
     if (orgUserCount !== nextQuantity) {
-      sendToSentry(new Error('Stripe Quantity Mismatch'), {
+      const errorStr = `Stripe Quantity Mismatch:
```

Comment:
we get 8192 chars (https://develop.sentry.dev/sdk/event-payloads/message/)
this will be truncated.
could you make a pg table & push there instead?